src/tools/tools/file.fs

```diff
@@ -228,17 +228,17 @@ module  FileTools =
         let mutable message = ""
         if File.Exists( filename ) then 
             bExist <- true 
-            else
-                Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: create and write file '%s'" filename)
-                try 
-                    use fileStream = CreateFileStreamForWrite( filename )
-                    fileStream.Write( bytes, offset, len )
-                    fileStream.Close() 
-                with 
-                | e -> 
-                    message <- e.Message
-                    bExist <- true
-                Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: complete create and write file '%s' (exists = %b)" filename bExist)
+        else
+            Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: create and write file '%s'" filename)
+            try 
+                use fileStream = CreateFileStreamForWrite( filename )
+                fileStream.Write( bytes, offset, len )
+                fileStream.Close() 
+            with 
+            | e -> 
+                message <- e.Message
+                bExist <- true
+            //Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: complete create and write file '%s' (exists = %b)" filename bExist)
```

Comment:
why this line was commented out? It was added intentionally to help track down an issue.